src/ds5/ds5-device.cpp

```diff
@@ -920,6 +920,17 @@ namespace librealsense
 
             }
 
+            if (_pid == RS_S585_PID) {
+                // SAFETY PRESET OPTION
+                // need to implement new safety preset option to handle range, and implement our set/query
+                auto active_safety_preset = std::make_shared<uvc_xu_option<uint8_t>>(raw_depth_sensor,
```

Comment:
The parameters that you will have to get from hkr team is:
size , curretly you placed `uint8_t`
and `DS5_ACTIVE_SAFETY_PRESET`

please remove the second comment, unless it is temporary just for you :)

Reply:
We will have a conflict here, once we merge the code we will align.
I added a special folder for registering safety camera option and a new namespace for the safety XU id's